opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -508,6 +517,61 @@ message DoubleHistogramDataPoint {
   repeated DoubleExemplar exemplars = 8;
 }
 
+// SummaryDataPoint is a single data point in a timeseries that describes the
```

Comment:
```suggestion
// DoubleSummaryDataPoint is a single data point in a timeseries that describes the
```

Reply:
I'm following the same convention on comments of the other data structures

Reply:
I see 
```
// IntDataPoint is a single data point in a timeseries that describes the
// time-varying values of a int64 metric.
message IntDataPoint {
```
and
```
// DoubleDataPoint is a single data point in a timeseries that describes the
// time-varying value of a double metric.
message DoubleDataPoint {
```
The convention should be to use the full name of the message in the comment. If elsewhere it is not so it needs to be fixed too. DoubleHistogramDataPoint appears to be violating the convention, can be fixed in a future PR.